### ISSUE TITLE
Add (Python-based) device hangs in 7.0.204 in traverser, mdstcl, and python

### ISSUE THREAD

**Issue by bstandley** [MDSplus/mdsplus#290]
In MDSTCL, the following hangs in the latest alpha:
edit treename
add node nodename /model=ACQ196

I think it is related to the new Py.py from commit 5280c4b.  7.0.203 is OK.

**Comment by zack-vii** [comment]
hm python directly fails as well:

``` python
import MDSplus
tree = MDSplus.Tree('test',-1,'new')
tree.addDevice('A','ACQ196')
```

**Comment by zack-vii** [comment]
fixed by #293
